### PR TITLE
Bug fix/aoo 29 wrong server name in hamburger menu

### PR DESCRIPTION
**Pull request by Elelan**
https://oadev.atlassian.net/browse/AOO-29

### CODE REVIEW

File: app/src/main/java/net/opendasharchive/openarchive/features/main/MainActivity.kt [hunk 1]
```diff
@@ -285,6 +279,16 @@ class MainActivity : BaseActivity(), FolderAdapterListener, SpaceAdapterListener
         }
     }
 
+    private fun updateCurrentSpaceAtDrawer() {
+        mBinding.currentSpaceName.text = Space.current?.friendlyName
+        mBinding.currentSpaceName.setDrawable(
+            Space.current?.getAvatar(applicationContext)?.scaled(32, applicationContext),
```
Comment (prathieshna) [inline app/src/main/java/net/opendasharchive/openarchive/features/main/MainActivity.kt:285]: This 32 should come from dimensions, isn't it?
Why are we sending application context?

**Review by prathieshna — CHANGES_REQUESTED** [review]
Please respond to the comments.

File: app/src/main/java/net/opendasharchive/openarchive/features/main/MainActivity.kt (hunk 1 above)
Comment (Elelan) [reply app/src/main/java/net/opendasharchive/openarchive/features/main/MainActivity.kt:285]: ApplicationContext is used in getAvatar function to retrieve Resource Colors
```kotlin
val color = ContextCompat.getColor(context, R.color.colorOnBackground)
```

**Review by prathieshna — APPROVED** [review]
LGTM

**Pull request merged**